modules/hcp-support-matrix.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * hosted-control-planes/index.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-support-matrix"_{context}"]
+= Support matrix for {hcp}
+
+Because {mce} includes the HyperShift Operator, releases of {hcp} align with releases of {mce-short}. For more information, see "OpenShift Operator Life Cycles".
+
+Any supported {product-title} cluster can be a management cluster. The following table maps {mce-short} versions to the management cluster versions that support them:
+
+.Supported {mce-short} versions for {product-title} management clusters
+[cols="2",options="header"]
+|===
+|Management cluster version |Supported {mce-short} version
+
+|4.12 - 4.15
+|2.4
+
+|4.13 - 4.16
+|2.5
+
+|4.14 - 4.17
+|2.6
+
+|4.15 - 4.17
+|2.7
+
+|===
+
+For hosted clusters, no direct relationship exists the management cluster version and the hosted cluster version. The hosted cluster version depends on the HyperShift Operator of your {mce-short} version.
+
+If you have a running hosted cluster and you update {mce-short} to a later version, provided that the HyperShift Operator supports the hosted cluster version, the hosted cluster is not affected.
+
+The following table indicates which {product-title} versions are supported for each platform of {hcp}. In the table, management cluster version refers to the {product-title} version where the {mce-short} is enabled:
+
+.Required {product-title} versions for platforms
+[cols="3",options="header"]
+|===
+|Platform |Management cluster version |Hosted cluster version
+
+|{aws-full}
+|4.12 - 4.17
+|4.14 - 4.17
+
+|{ibm-power-title}
+|4.17
+|4.17
+
```

Comment:
This was possible as TP before 4.17

Reply:
Updated to "4.15 - 4.17", as I believe 4.15 was the first release where IBM Power was TP.